airflow/contrib/utils/log/task_handler_with_custom_formatter.py

```diff
@@ -25,10 +28,20 @@
 
 
 class TaskHandlerWithCustomFormatter(StreamHandler):
+    """
+
+    Custom implementation of StreamHandler, a class which writes logging records for Airflow
+    """
     def __init__(self, stream):
         super().__init__()
+        self.prefix_jinja_template = None
 
     def set_context(self, ti):
+        """
+        Accept the run-time context (i.e. the current task) and configure the formatter accordingly.
+        :param ti:
```

Comment:
```suggestion

         :param ti:
```

(This typed on phone, so double check spacing

Otherwise docs don't format right